### PR TITLE
Record module view

### PR DESCRIPTION
**Pull request by Chahlyk**

### CODE REVIEW

File: src/app/pages/record/category-table/category-table.component.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<div class="mat-elevation-z8">
```
Comment (Liliia-Abr) [inline src/app/pages/record/category-table/category-table.component.html]: this is class from example

File: src/app/pages/record/record.component.ts [hunk 2]
```diff
@@ -1,15 +1,37 @@
...
 @Component({
   selector: 'app-record',
   templateUrl: './record.component.html',
   styleUrls: ['./record.component.css']
 })
-export class RecordComponent implements OnInit {
+export class RecordComponent implements OnInit, OnDestroy {
 
-  constructor() { }
+  public sub: Subscription = new Subscription();
```
Comment (Liliia-Abr) [inline src/app/pages/record/record.component.ts]: why is it public?

File: src/app/pages/record/record.module.ts [hunk 3]
```diff
@@ -2,18 +2,22 @@ import { NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import {RecordComponent} from './record.component';
```
Comment (Liliia-Abr) [inline src/app/pages/record/record.module.ts]: style code

File: src/app/pages/record/record.module.ts [hunk 4]
```diff
@@ -2,18 +2,22 @@ import { NgModule } from '@angular/core';
...
+    CategoryTableComponent
   ],
   exports: [
     RecordComponent
   ],
-  imports: [
-    CommonModule,
-    RecordRoutingModule
-  ]
+    imports: [
```
Comment (Liliia-Abr) [inline src/app/pages/record/record.module.ts:17]: style code, check spaces

File: src/app/pages/record/category-table/category-table.component.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+  <table mat-table [dataSource]="dataSource">
+
+    <ng-container matColumnDef="id">
+      <th mat-header-cell *matHeaderCellDef>No.</th>
+      <td mat-cell *matCellDef="let element"> {{element.id}} </td>
```
Comment (Liliia-Abr) [inline src/app/pages/record/category-table/category-table.component.html]: it's not id, it's index

File: src/app/pages/record/category-table/category-table.component.html [hunk 6]
```diff
@@ -0,0 +1,35 @@
+<div>
+  <table mat-table [dataSource]="dataSource">
+
+    <ng-container matColumnDef="index">
+      <th mat-header-cell *matHeaderCellDef>No.</th>
+      <td mat-cell *matCellDef="let element"> {{dataSource.indexOf(element) + 1}} </td>
```
Comment (Liliia-Abr) [inline src/app/pages/record/category-table/category-table.component.html]: it can be more simple

Comment (Chahlyk) [reply src/app/pages/record/category-table/category-table.component.html]: I have no idea how to simplify

**Review by Liliia-Abr — APPROVED** [review]

**Pull request merged**